src/main/java/de/retest/recheck/auth/RetestAuthentication.java

```diff
@@ -119,7 +120,10 @@ public void login( final AuthenticationHandler handler ) throws IOException, Htt
 				handler.authenticationFailed( callback.result.getErrorException() );
 			}
 
-			processCode( callback.result.getCode(), redirectUri );
+			final AccessTokenResponse tokenResponse =
+					ServerRequest.invokeAccessCodeToToken( deployment, callback.result.getCode(), redirectUri, null );
+			offlineToken = tokenResponse.getRefreshToken();
```

Comment:
Aber KeyCloak nennt das `refreshToken`. Sollten wir es nicht genauso nennen, um Mißverständnisse zu vermeiden?

Reply:
There is no real offline token. These are all refresh tokens.

Reply:
So we should rename the field.